src/components/home/elements/BurnToken.vue

```diff
@@ -0,0 +1,750 @@
+    data: function() {
+        return {
+            amount: BigNumber(0),
+            coldAmount: BigNumber(0),
+            attachment: '',
+            pageId: 1,
+            fee: BigNumber(CONTRACT_EXEC_FEE),
+            coldPageId: 5,
+            coldFee: BigNumber(CONTRACT_EXEC_FEE),
+            coldAddress: this ? (this.walletType === 'coldWallet' ? this.selectedAddress : this.defaultColdAddress) : '',
+            scanShow: false,
+            sendError: false,
+            coldSignature: '',
+            timeStamp: Date.now() * 1e6,
+            hasConfirmed: false
+        }
+    },
+    props: {
+        tokenBalance: {
+            type: BigNumber,
+            default: function() {
+                return BigNumber(0)
+            },
+            require: true
+        },
+        tokenUnity: {
+            type: BigNumber,
+            default: function() {
+            },
+            require: true
+        },
+        coldAddresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        addresses: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        balance: {
+            type: Object,
+            default: function() {},
+            require: true
+        },
+        address: {
+            type: String,
+            default: ''
+        },
+        walletType: {
+            type: String,
+            default: 'hotWallet',
+            require: true
+        },
+        tokenId: {
+            type: String,
+            default: '',
+            require: true
+        },
+        issuer: {
+            type: String,
+            default: '',
+            require: true
+        }
+    },
+    computed: {
+        contractId() {
+            return transaction.tokenIDToContractID(this.tokenId)
+        },
+        defaultAddress() {
+            return Vue.ls.get('address')
+        },
+        defaultColdAddress() {
+            if (this.noColdAddress) return ''
+            return Object.keys(this.coldAddresses)[0]
+        },
+        userInfo() {
+            return JSON.parse(window.localStorage.getItem(this.defaultAddress))
+        },
+        secretInfo() {
+            return JSON.parse(
+                seedLib.decryptSeedPhrase(this.userInfo.info, Vue.ls.get('pwd')))
+        },
+        seedPhrase() {
+            return seedLib.decryptSeedPhrase(this.secretInfo.encrSeed, Vue.ls.get('pwd'))
+        },
+        wordList() {
+            return this.seedPhrase.split(' ')
+        },
+        isSubmitDisabled() {
+            return !(BigNumber(this.amount).isGreaterThan(0) && this.isValidIssuer(this.address) && (this.isValidAttachment || !this.attachment) && this.isAmountValid('hot'))
+        },
+        isColdSubmitDisabled() {
+            return !(this.coldAmount > 0 && this.isValidIssuer(this.address) && (this.isValidColdAttachment || !this.coldAttachment) && this.isAmountValid('cold'))
```

Comment:
any difference between `BigNumber(this.amount).isGreaterThan(0)` and `this.coldAmount > 0`?
why use different check here?